tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml

```diff
@@ -465,78 +499,103 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
  <section title="Controller Plane (Management and Control) Considerations" anchor="cp_considerations">
 
    <t>
-	[Editor's note: This section covers management/control plane related
-	implications of creation, mapping, removal of DetNet flow IDs, their
-	related parameters and, when needed, the configuration of PREOF.]
+        TSN Stream(s) to DetNet flow mapping related information are
+        required only for the service proxy function of MPLS (DetNet) Edge nodes. 
+        From the Data Plane perspective TSN flows are application flows for 
+        DetNet. The proxy behaves as an Application interface for DetNet and 
+        as a TSN relay node for TSN Ethernet flows.         
    </t>
    <t>
-	TSN Stream(s) to DetNet flow mapping related information are
-	required only for the service proxy function of MPLS (DetNet) Edge nodes. 
-	From the Data Plane perspective there is no practical difference
-	based on the origin of flow mapping related information	(management plane 
-	or control plane).
+        MPLS DetNet Edge nodes are member of both the DetNet domain and the
+        connected TSN network. From the TSN network perspective the MPLS
+        (DetNet) Edge node has a "TSN relay node" role, so TSN specific
+        management and control plane functionalities must be implemented.
+        There are many similarities in the management plane techniques used in
+        DetNet and TSN, but that is not the case for the control plane
+        protocols. For example, RSVP-TE and MSRP behaves differently.  Service
+        interworking is out of scope for this document.  TSN services simply
+        see the DetNet network as a TSN relay node that supports forwarding to
+        the MAC address in the TSN flow. 
    </t>
    <t>
-	MPLS DetNet Edge nodes are member of both the DetNet domain and the 
-	connected TSN network. From the TSN network perspective the MPLS (DetNet) 
-	Edge node has a "TSN relay node" role, so TSN specific management and
-	control plane functionalities must be implemented.  There are many 
-	similarities in the management plane techniques used in DetNet and TSN, 
-	but that is not the case for the control plane protocols. For example, 
-	RSVP-TE and MSRP behaves differently. Therefore management and control
-	plane design is an important aspect of scenarios, where	mapping between 
-	DetNet and TSN is required.
+        The service Proxy is configured as a DetNet application and all functions are 
+        as described in <xref target="I-D.ietf-detnet-mpls"/>. As far as 
+        the TSN Network is concerned the service proxy is a TSN relay node that supports
+        one or more of TSN destination MAC addresses. These MAC address may be unicast or
+        multicast but the Detnet domain service proxy supports only one type at a time.
+        The service proxy is configured as a unicast or multicast Application for the DetNet 
+        interface.   

```

Comment:
"I-D.ietf-detnet-mpls" does not defines the proxy function. I have also concerns of such terminology "TSN destination MAC address". I think you mean here the "dst MAC address of the TSN stream".